### PR TITLE
scANVI: return un-propagated raw logits

### PR DESCRIPTION
**Pull request by sjfleming**
Have `predict()` return the raw logits as `"cell_logits_nc"` and changed SOCAM to use the same name.  This is for compatibility with CAS, so CAS can know what to expect.

### CODE REVIEW

**Comment by sjfleming** [comment]
I think I'm just gonna close this in favor of #424 , since that includes these changes
